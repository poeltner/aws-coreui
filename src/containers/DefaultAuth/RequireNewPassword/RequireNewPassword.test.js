/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import DefaultRequireNewPassword from './RequireNewPassword';

describe('RequireNewPassword', () => {
  describe('normal case', () => {
    test('render correctly', () => {
      const wrapper = shallow(<DefaultRequireNewPassword />);

      wrapper.setProps({
          authState: 'requireNewPassword',
          authData: {
              challengeParam: {
                  requiredAttributes: []
              }
          },
          hide: false
      });

      expect(wrapper).toMatchSnapshot();
    });
  });
});