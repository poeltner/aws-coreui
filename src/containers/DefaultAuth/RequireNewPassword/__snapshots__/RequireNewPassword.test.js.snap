// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RequireNewPassword normal case render correctly 1`] = `ShallowWrapper {}`;
